src/network/networkd-dhcp4.c

```diff
@@ -632,13 +637,24 @@ int dhcp4_configure(Link *link) {
         }
 
         /* Always acquire the timezone and NTP */
-        r = sd_dhcp_client_set_request_option(link->dhcp_client, SD_DHCP_OPTION_NTP_SERVER);
-        if (r < 0)
-                return r;
+        /* NOTE: why this assertion? ^^^ */
```

Comment:
i meant about the comment <<Always acquire the timezone and NTP>>,
as they won't be sent using Anonymize.